src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/BundleViewPathPass.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Config\Resource\FileExistenceResource;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Registers the bundles view paths.
+ */
+final class BundleViewPathPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        $twigFilesystemLoaderId = 'twig.loader.native_filesystem';
```

Comment:
Should we use `twig.loader.filesystem` instead + `findDefinition()`? Ref:
https://github.com/symfony/symfony/blob/2ac5f1d08bc69dd47179080d40c769fb7af2ad0d/src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php#L99-L108

Otherwise, this compiler pass must run before `ExtensionPass`, where the native filesystem loader can be turned into `twig.loader.filesystem` when the Templating component is activated. (LINE 102)

Reply:
yes, done !